ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/templates/hclecomm-content-page-template/policies/.content.xml

```diff
@@ -32,10 +32,14 @@
                                 cq:policy="hclecomm/components/content/accordion/policy_1588258216814"
                                 jcr:primaryType="nt:unstructured"
                                 sling:resourceType="wcm/core/components/policies/mapping"/>
-							<container
+							              <container
```

Comment:
@rakeshkumar-verma : can we remove the extra space here ?

Reply:
@rishabh-behl  Removed extra space.